docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,87 @@
+Run the workflow using example data
+===================================
+
+In this tutorial, you will run the workflow using example reference data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`setup`. These instructions will install all of the software you need to complete this tutorial and others.
+
+Setup
+-----
+
+1. Activate the ``nextstrain`` conda environment:
+
+   .. code:: text
+
+      conda activate nextstrain
+
+2. Change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+3. Download the example tutorial repository into a new directory ``ncov-tutorial/``:
+
+   .. code:: text
+
+      git clone https://github.com/nextstrain/ncov-tutorial
```

Comment:
Can we add `ncov-tutorial` to the `.gitignore` in the ncov repo? This would avoid the following situation

```
$ git status
On branch tutorial
Your branch is up to date with 'origin/tutorial'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        ncov-tutorial/

nothing added to commit but untracked files present (use "git add" to track)
```

Reply:
Done: fbdb4d58

Reply:
Updated to ignore `my-analyses/` according to new instructions (see https://github.com/nextstrain/ncov/pull/894#discussion_r840122613)